src/tutorial/runtime/introduction.rst

```diff
@@ -185,51 +178,62 @@ The Cylc GUI
    interface (the Cylc GUI) which can be used for monitoring and
    interaction.
 
-   The Cylc GUI looks quite like ``cylc graph`` but the tasks are colour-coded
-   to represent their state, as in the following diagram.
+   The Cylc UI has different views you can use to examine your workflows.
 
-.. digraph:: example
-   :align: center
+- Task states have grey icons.
+- Job states have colour-coded squares.
 
-   Waiting [color="#88c6ff"]
-   Running [style="filled" color="#00c410"]
-   Succeeded [style="filled" color="#ada5a5"]
+.. csv-table::
+   :header: Common task and job states, Description
 
-.. minicylc::
-   :align: center
+   |task-waiting|, Task waiting on other tasks
+   |task-submitted| |job-submitted|, Job submitted
+   |task-running| |job-running|, Job is running
+   |task-succeeded| |job-succeeded|, Job has run successfully
+   |task-failed| |job-failed|, Job failed
 
-    a => b => c
-    b => d => f
-    e => f
+.. ifnotslides::
+
+   .. seealso::
+
+      Full list of :ref:`task-job-states`.
 
 .. nextslide::
 
 .. ifnotslides::
 
-   This is the "graph view". The Cylc GUI has two other views called "tree" and
-   "dot".
+   This is the "tree" view:
 
-.. figure:: ../img/cylc-gui-graph.png
+.. figure:: ../img/cylc-gui-tree-view.png
    :figwidth: 75%
    :align: center
 
-   Screenshot of the Cylc GUI in "Graph View" mode.
+   Screenshot of the Cylc GUI in "Tree" view mode.
 
 .. nextslide::
 
-.. figure:: ../img/cylc-gui-tree.png
+This is the "table" view:
+
+.. figure:: ../img/cylc-gui-table-view.png
    :figwidth: 75%
    :align: center
 
-   Screenshot of the Cylc GUI in "Tree View" mode.
+   Screenshot of the Cylc GUI in "Table" view mode.
 
 .. nextslide::
 
-.. figure:: ../img/cylc-gui-dot.png
+.. ifnotslides::
+
+   There is a GUI "scan" (also known as "GScan") view on the left allowing you
+   to navigate your workflows.
+
+.. TODO - re-do this figure when UI confusing elements removed.
```

Comment:
?

Link to issue?

Reply:
This: https://github.com/cylc/cylc-ui/issues/881

and

I can't find the ticket, but I think that we were going to remove this at some point:
![image](https://user-images.githubusercontent.com/26465611/151994826-110344b2-cd59-443f-a24d-8456b1774e7a.png)

Reply:
> (Would love to auto-generate our screenshots, which I think is perfectly possible to do with Cypress!)

Nice